src/rust/ide/src/double_representation/graph.rs

```diff
@@ -430,7 +430,7 @@ main =
         println!("zz");
```

Comment:
Why do we have this print? I know it isn't part of this PR, but it just caught my attention.

Reply:
I remove those in my next PR.